rasa_nlu/classifiers/regex_intent_classifier.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from builtins import map
+from typing import Any
+from typing import Dict
+from typing import Text
+import re
+
+from rasa_nlu.components import Component
+
+
+class RegExIntentClassifier(Component):
+
+    name = "intent_classifier_regex"
+
+    context_provides = {
+        "process": ["intent"],
+    }
+
+    output_provides = ["intent"]
+
+    def __init__(self, clf=None, regex_dict=None):
+        self.clf = clf
+        self.regex_dict = {}
+
+    def train(self, training_data):
```

Comment:
Ah! Hadn't seen that training_data was its own data type... So should the regexp should be in the "text" part of the training data file?